### ISSUE TITLE
Data 

### ISSUE THREAD

**Issue by iexermr** [WorldChanger01/CODIGEM#1]
Thank you for the open-source code, but I have a question： Why are the data in the validation set and test set divided into two parts: training and testing? Why does the data of the evaluate function not use the training set? thanks in advance.
